app/routes/sites/companies/company/network-sites/new.js

```diff
@@ -0,0 +1,35 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  actions: {
+    submit(data) {
+      const companyId = this.paramsFor('sites.companies.company').id;
```

Comment:
you can use `const { id: companyId } = this.paramsFor('sites.companies.company')` (I believe)